edge/pkg/edgehub/process.go

```diff
@@ -154,15 +154,15 @@ func (eh *EdgeHub) routeToEdge() {
 		}
 		message, err := eh.chClient.Receive()
 		if err != nil {
-			klog.Errorf("websocket read error: %v", err)
+			klog.Errorf("EdgeHub websocket read error: %v", err)
 			eh.reconnectChan <- struct{}{}
 			return
 		}
 
-		klog.Infof("received msg from cloud-hub:%+v", message)
+		klog.V(2).Infof("EdgeHub received msg from cloud-hub: %+v", message)
```

Comment:
why use V2?

Reply:
Message here will be logged by MetaManager or other modules as well. For my view, edgehub would better be transparent and only to log error.